doc/introduction/remote.rst

```diff
@@ -197,17 +123,17 @@ Next we create the program
 
         ops.MeasureFock() | q
 
-We create the engine. The engine is in charge of compiling and executing
-programs on the remote device.
+Finally, we create the engine. Similarly to the :class:`~.LocalEngine`, the :class:`~.RemoteEngine`
+is in charge of compiling and executing programs, however it differs in that the program will be
+executed on *remote* devices, rather than on local simulators.
 
->>> eng = RemoteEngine("chip2")
+>>> eng = RemoteEngine("X8_01")
 
 We run the engine by calling ``eng.run``, and pass it the program we
```

Comment:
Better to phrase as running the program rather than running the engine

Reply:
Rephrase this to:
> We can now run the program by calling ``eng.run``, and passing the program to be executed
as well as additional runtime options.